packages/core/src/lib/migrations.ts

```diff
@@ -17,11 +17,13 @@ import { confirmPrompt, textPrompt } from './prompts';
 // We also want to silence messages from Prisma about available updates, since the developer is
 // not in control of their Prisma version.
 // https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_hide_update_message
-function runMigrateWithDbUrl<T>(dbUrl: string, cb: () => T): T {
+function runMigrateWithDbUrl<T>(dbUrl: string, shadowDbUrl: string | undefined, cb: () => T): T {
   let prevDBURLFromEnv = process.env.DATABASE_URL;
+  let prevShadowDBURLFromEnv = process.env.SHADOW_DATABASE_URL;
   let prevHiddenUpdateMessage = process.env.PRISMA_HIDE_UPDATE_MESSAGE;
   try {
     process.env.DATABASE_URL = dbUrl;
+    process.env.SHADOW_DATABASE_URL = shadowDbUrl;
```

Comment:
Might need to only set `process.env.SHADOW_DATABASE_URL` if `shadowDbUrl` is set.
ie
```
if (shadowDbUrl) {
    process.env.SHADOW_DATABASE_URL = shadowDbUrl;
}
```
otherwise it looks like ` process.env.SHADOW_DATABASE_URL` gets set to `undefined` which prisma doesn't seem to like.